app/views/posts/show.html.erb

```diff
@@ -17,14 +17,30 @@
         <%= form.submit 'Like Post' %>
       <% end %>
     </div>
+    <% if can?(:destroy, @post)%>
+          <%= link_to user_post_path(@user, @post), data: {
+            turbo_method: :delete
+          } do %>
+            <button><%= ('.delete post') %></button>
+            <% end %>
+        <% end %>
    <div class="post_comment">
    <% @post.comments.includes(:author).each do |comment| %>
       <ul class="comments width"> 
           <li> <%= comment.author.name %>: <%= comment.text%> </li>
        </ul>
-
     <% end %>
    </div>
+  
+  <div class="destroycomment"> 
+  <% if can?(:destroy, comment)%>
+   <%= link_to user_post_comment_path(@user, comment.post, comment.id), data: {
+     turbo_method: :delete
```

Comment:
Goodjob working on the project requirements whenever I try and create a comment I am encountering the following error.

![Screenshot 2023-05-25 at 12 02 49 PM](https://github.com/hakichris/Blog-App/assets/79506105/55bb1989-2251-4f41-a46f-5b1a15adaefe) 

Im also supposed to be able to like and delete a comment or a post but its not functional in your app.
It would be nice to fix this issue

Reply:
Thank you @KayLemba, I fixed that.